packages/record/src/index.js

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
here we can remove this eslint rule exception since it is only needed when there is only one named export